src/app/header/header.component.html

```diff
@@ -6,13 +6,31 @@
             <a class="btn btn-primary" routerLink="/" (click)="logout()">LOGOUT</a>
         </div>
 </div> -->
-
-<mat-toolbar color="primary" class="nav-bar">
-  <div>BINGEWATCH</div>
-  <div>
-    <span>Hello {{ userName | titlecase }}</span>
-    <a *ngIf="adminAccess" routerLink="/checkin">Checkin Scanner</a>
-    <a routerLink="/mybookings">My Bookings</a>
-    <a (click)="logout()">Logout</a>
-  </div>
-</mat-toolbar>
+<div class="container">
+    <mat-toolbar color="primary" class="nav-bar">
```

Comment:
Component Naming doesn't makes sense.
Header component having Body component inside it.